### DISCUSSION TITLE
Any consequences using this program?

### DISCUSSION THREAD

**Discussion by gprog002** [kubilus1/autoortho#151] [General]
Will/ can map providers ban my IP after some time if I use AO too much, I mean every flight and download lots of data? I read somewhere ARC will ban by IP, if you use it too much. Or can we use these services without any consequences?

**Comment by BinaryOutlook** [comment]
I got 88gb cached, yet still haven't been banned.

**Comment by Glasairmell** [comment]
Just in case I am using a VPN when I know there is a large download.
